src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -1522,6 +1522,286 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe Vector<ulong> DuplicateSelectedScalarToVector(Vector<ulong> data, [ConstantExpected(Min = 0, Max = (byte)(7))] byte index) => DuplicateSelectedScalarToVector(data, index);
 
 
+        /// <summary>
+        /// uint8_t svlasta[_u8](svbool_t pg, svuint8_t op)
+        ///   LASTA Wresult, Pg, Zop.B
+        ///   LASTA Bresult, Pg, Zop.B
+        /// </summary>
+        public static unsafe byte ExtractAfterLastScalar(Vector<byte> value) => ExtractAfterLastScalar(value);
```

Comment:
The issue is that the managed signature of these APIs isn't correct (and wasn't caught in API review).

These all behave specially and need to take a mask explicitly, so should've been in a general shape that looked more like:
```csharp
public static T ExtractAfterLastActiveToScalar(Vector<T> mask, Vector<T> value)
public static Vector<T> ExtractAfterLastActiveToVector(Vector<T> mask, Vector<T> value)
```

These are shipping as experimental in .NET 9, so we can go ahead and fix it now and follow up in API review later, but we should log an issue tracking that.

The general consideration is that `LASTA` and `LASTB` take a `destination register`, `mask register`, and `input register`. The mask is used to determine which input element is being extracted, so for `LASTA` (extract after last) it finds the index of the last element in `mask` that is "active" (all bits set) and extracts `index+1` from `value`. In the case that the mask is `zero` or `allbitsset`, it uses index 0. While for `LASTB` (extract last) it finds the index of the last element in `mask` that is "active" (all bits set) and extracts that `index` from `value`. In the case that mask is `zero` or `allbitsset`, it uses the last index.

So, the signatures must explicitly take a mask and they shouldn't participate in the general `EmbeddedMaskOperation` containment logic

Reply:
-- The names I gave above maybe aren't the best.

The concepts involved is we have what is logically similar to:
* `T Extract(Vector<T> vector, int index)`
* `Vector<T> ExtractVector(Vector<T> vector, int index)`

There's then a modifier on top which in other APIs is exposed as `LastActiveElement` and `AfterLastActiveElement`

So the natural way to combine them would maybe instead be:
```csharp
T ExtractAfterLastActiveElement(Vector<T> mask, Vector<T> vector);
Vector<T> ExtractVectorAfterLastActiveElement(Vector<T> mask, Vector<T> vector);

T ExtractLastActiveElement(Vector<T> mask, Vector<T> vector);
Vector<T> ExtractVectorLastActiveElement(Vector<T> mask, Vector<T> vector);
```

Reply:
Not super happy with `ExtractVector*` though since it's not actually extracting a vector

It's extracting a scalar and simply storing it in element 0 of a destination vector register

Reply:
Give that the vector form always zero extends and there isn't a great name for it, we could support it by recognizing `CreateScalar(ExtractFromLastActiveElement(mask, vector))`

That gives us a single understandable name and lets us generate the vector form where its appropriate still.